Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -434,6 +450,8 @@ open class ChatMessageListVC: _ViewController,
         cell.setDecoration(for: .header, decorationView: delegate?.chatMessageListVC(self, headerViewForMessage: message, at: indexPath))
         cell.setDecoration(for: .footer, decorationView: delegate?.chatMessageListVC(self, footerViewForMessage: message, at: indexPath))
 
+        // TODO: Add logic to show/hide separator based on `firstUnreadMessageId`
```

Comment:
This logic should be in the ChatChannelVC, right?

Reply:
After the rebase with develop, yeah! Did not see that your changes were applied 😄